### PR TITLE
Add Hawaiian names

### PR DESCRIPTION
**Pull request by zonkmachine**
Aloha!

### CODE REVIEW

File: data/culture/haw.lua [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+	'Kimo',
+	'Kimokea',
+	'Koa',
+	'Koalani',
+	'Konane',
+	'Kulani',
+	'Ky',
+	'Lihau',
+	'Lono',
+	'makai',
```
Comment (Web-eWorks) [inline data/culture/haw.lua:68]: Wrong case for a Proper Noun.

File: data/culture/haw.lua [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+	'Kulani',
+	'Kuʻuipo',
+	'Ky',
+	'Lana',
+	'Lei',
+	'Leinani',
+	'Lihau',
+	'Lilo', -- ;)
+	'Lono',
+	'luana',
```
Comment (Web-eWorks) [inline data/culture/haw.lua:139]: Wrong case.

File: data/culture/haw.lua [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+	'Keliʻi',
+	'Keona',
+	'Konani',
+	'Lani',
+	'Lee',
+	'Leilani',
+	'Leimomi',
+	'Leo',
+	'Likeke',
+	'Lilo',
```
Comment (Web-eWorks) [inline data/culture/haw.lua:230]: Note that this could create a name e.g. 'Lilo Lilo'. Is this well formed?

**Comment by impaktor** [comment]
I assume you've tested it in game by setting weight to something very high so everyone will become Hawaiian?

File: data/culture/haw.lua (hunk 3 above)
Comment (zonkmachine) [reply data/culture/haw.lua:230]: Yes. I believe this is how it's done.

**Comment by zonkmachine** [comment]
> I assume you've tested it in game by setting weight to something very high so everyone will become Hawaiian?

I sure have. A lot of testing to make sure the special signs are not shown.

**Comment by zonkmachine** [comment]
Many Hawaiian names are compound names and it's tempting to make a generator for it but I pass for now. It looks simple enough but it's for another day.

It's tempting to add a 'phrases' section to the language files. So the character could go randomly Aloha! in game.

Thanks for the review! I think this is ready to merge.

**Pull request merged**